src/common/Screen.js

```diff
@@ -69,6 +70,7 @@ type Props = $ReadOnly<{|
  * @prop [autoFocus] - If search bar enabled, should it be focused initially.
  * @prop [searchBarOnChange] - Event called on search query change.
  * @prop [searchBarOnSubmit] - Event called on search query submit.
+ * @prop [placeholder] - text shown as place holder, required if `search` is true.
```

Comment:
It's not true that this is required in that case, is it? After all, we don't pass it in any of the other places where we use `Screen` with `search` true.

(You can find those other places with e.g. `git grep searchBarOn`, because each such screen will want to use at least one of the callbacks `searchBarOnChange` and `searchBarOnSubmit`.)

Reply:
Let's call this prop `searchPlaceholder` here on `Screen`, to help make clear what it's a placeholder for.

OTOH for `SearchInput`, just `placeholder` is an appropriate name.